src/components/common/HotDeals/HotDealsHover/HotDealsHover.js

```diff
@@ -0,0 +1,41 @@
+import { faShoppingBasket } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Button from '../../Button/Button';
+import React from 'react';
+import styles from './HotDealsHover.module.scss';
+const HotDealsHover = () => {
```

Comment:
Lepsza nazwa by byłą `HotDealsIcons` hover to zachowanie które w przyszłości moze ulec zmianie